R/db-postgres.r

```diff
@@ -67,6 +67,38 @@ sql_translate_env.PostgreSQLConnection <- function(con) {
       },
       str_detect  = function(string, pattern){
         sql_expr(strpos(!!string, !!pattern) > 0L)
+      },
+      # lubridate functions
+      wday = function(x, label=FALSE, abbr = TRUE
+                      , week_start = getOption("lubridate.week.start",7)
+                      , locale = Sys.getlocale("LC_TIME")) {
+        if (!label) {
+          return(build_sql(
```

Comment:
Can you please eliminate the `return()`s and switch to `sql_expr()`

Reply:
It's useful for me to include a comment that points to the postgres docs of the function you're using (i.e. `extract()`)

Reply:
It might be easier to start without supporting the `label` or `abbr` arguments.